client/state/inline-help/test/selectors.js

```diff
@@ -135,3 +138,93 @@ describe( '#getInlineHelpSearchResultsForQuery()', () => {
 		expect( getInlineHelpSearchResultsForQuery( state, 'foo' ) ).to.deep.equal( [] );
 	} );
 } );
+
+describe( 'Admin section search and filters', () => {
+	describe( 'getAdminSectionsResults()', () => {
+		test( 'should return `[]` when there is no search term', () => {
+			const result = getAdminHelpResults( {} );
+			expect( result ).to.deep.equal( [] );
+		} );
+
+		test( 'should return admin help results for `domain` term', () => {
+			const results = getAdminHelpResults( {}, 'Add a new domain' );
```

Comment:
I think it would be safe here to asset against the _actual_ results being returned, rather than simply checking you don't have an empty result set. The test outline itself suggests you are asserting against the presence of something, not the absence so the test body should reflect that.

I would probably be ok to assert against real results being returned for "domain" as they are hard coded and therefore won't change. Alternatively, you could mock out the list before asserting.

Reply:
> I think it would be safe here to asset against the actual results being returned, rather than simply checking you don't have an empty result set.

Let me find a way to test it.

Reply:
The thing is it isn't so easy to compare only one result because `getAdminHelpResults()` returns many items (it's an array). So we could re-define the query search in order to reduce the results to one item and then compare the object, but it sounds fragile to me since we'll rely on the admin sections data. If something changes there, the tests will break.
Can we hold on this and iterate later instead? I'm going to feel more confident about the API once the entire implementation develops.

Reply:
because it's pretty probable we are going to move these data to global state tree. On that case, we can mock the testing data.